test/v1/controllers/requestParsers/RetrievePeriodSummaryRequestParserSpec.scala

```diff
@@ -49,7 +57,12 @@ class RetrievePeriodSummaryRequestParserSpec extends UnitSpec {
         MockRetrievePeriodSummaryValidator.validate(rawData).returns(Nil)
 
         parser.parseRequest(rawData) shouldBe
-          Right(RetrievePeriodSummaryRequest(Nino(nino), BusinessId(businessId), periodId))
+          Right(RetrievePeriodSummaryRequest(Nino(nino), BusinessId(businessId), PeriodId(periodId), None))
+      }
+      "valid TYS request data is supplied" in new Test {
```

Comment:
Do we need TYS-specific tests for requestParser and validator? They weren't added in  individuals-income-received-api which was to use for reference (not saying this is wrong though).

Reply:
Because we have tax year as a param here so we should add a test for request parser and validator